scipy/io/idl.py

```diff
@@ -674,14 +674,14 @@ def __setitem__(self, key, value):
 def readsav(file_name, idict=None, python_dict=False,
             uncompressed_file_name=None, verbose=False):
     """
-    Read an IDL .sav file
+    Read an IDL .sav file.
```

Comment:
I like this change for consistency, but I'm curious how this was breaking the parsing? Was it in some introspection tool (like Spyder)?